dashboard/test/controllers/registrations_controller/set_user_type_test.rb

```diff
@@ -23,9 +23,6 @@ class SetUserTypeTest < ActionDispatch::IntegrationTest
       assert_response :bad_request
     end
 
-    #
-    # Tests for non-migrated users
-    #
```

Comment:
Did you opt to remove the non-migrated user tests here? I can't wait to do this, but it seems like we're not _quite_ ready to do so until we've migrated everyone.

Reply:
I did! Although looking at this again, I can see how these tests are definitely still valuable. I'll restore em, and just update them all to use `demigrated`